doc/sphinxext/ipython_sphinxext/ipython_directive.py

```diff
@@ -127,7 +127,11 @@
 from sphinx.util.compat import Directive
 
 # Our own
-from IPython import Config, InteractiveShell
+try:
+    from traitlets.config import Config
+except ImportError:
```

Comment:
Apologies for the delayed response. I think it's an oversight that we forgot to shim for the top level module, but now that it's out there without the shim, we're probably not going to go back and add new shims, so this fix should be correct anyway.